app/src/main/AndroidManifest.xml

```diff
@@ -49,8 +48,10 @@
         <activity
             android:name=".SearchMapsActivity"
             android:label="@string/title_activity_search_maps" />
-        <activity android:name=".SearchActivity"></activity>
->>>>>>> refs/remotes/origin/master
+        <activity android:name=".SearchActivity" />
+        >>>>>>> refs/remotes/origin/master
```

Comment:
I think Git put that in there...I'm not sure what that means either or where that came from...